script/install-package-tensorflowmodel-object-detection/install.sh

```diff
@@ -23,8 +23,13 @@ rm -rf tmp
 
 #####################################################################
 echo ""
-echo "Copy label-map file ..."
-cp -f ${CK_ENV_TENSORFLOW_MODELS_OBJ_DET_DIR}/data/${LABELMAP_FILE} .
+#old version, based on the labelmaps from tf api folder
+#echo "Copy label-map file ..."
```

Comment:
Echo would still be useful, but the old cmd should be removed rather than commented out.